packages/wicked/1007-include-mac-address-in-leaseinfo.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/src/leaseinfo.c b/src/leaseinfo.c
+index c47241f5..95f7fb0e 100644
+--- a/src/leaseinfo.c
++++ b/src/leaseinfo.c
+@@ -692,6 +692,18 @@ __ni_leaseinfo_dump(FILE *out, const ni_addrconf_lease_t *lease,
+ {
+ 	unsigned int i;
+ 	char *key = NULL;
++	ni_netconfig_t *nc = ni_global_state_handle(0);
```

Comment:
There is prior art for querying a device in [update.c](https://github.com/openSUSE/wicked/blob/5e50cdbb5d74490a9d728c0d0be537985bb65b38/src/update.c#L503-L511) which is actually where `leaseinfo_dump()` is called.